e2e-tests/build-and-run

```diff
@@ -2,11 +2,23 @@
 
 set -o errexit
 
+nobuild=0
+if [[ $# -ge 1 ]]; then
+  if [[ ( $1 = "-nb" ) || ( $1 == "--nobuild") ]]; then
+     shift
+     nobuild=1
+  fi
+fi
+
 test_dir="$(dirname $0)"
 . $(dirname $0)/functions
 src_dir="$(realpath $test_dir/..)"
 
-$test_dir/build
+if [[ $nobuild -eq 0  ]]; then
+    echo "building, use -nb or --nobuild to suppress this"
+    $test_dir/build || exit
+fi
+
```

Comment:
could we unify approach for both operators, in PXC we do
```
BUILD=0 ./e2e-tests/build-and-run
```
code - https://github.com/percona/percona-xtradb-cluster-operator/blob/master/e2e-tests/build#L45-L47

Reply:
I like `-nb` flag much more than setting `BUILD` env variable, but if you insist I can change it.

Reply:
I am ok, on any approach, but please let it be identical in both products

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/300/commits/b26bb9b5c7917ece560211316a039dbc0747f297